UPGRADE-4.4.md

```diff
@@ -269,7 +269,21 @@ TwigBridge
 TwigBundle
 ----------
 
- * Deprecated `twig.exception_controller` configuration option, set it to "null" and use `framework.error_controller` instead:
+ * Deprecated `twig.exception_controller` configuration option.
+
+   If you were not using this option previously, set it to `null`:
+   Before:
+   ```yaml
+   twig:
+   ```
```

Comment:
I think you can remove this "before" empty example, IMO the message is already clear about the previous use. wdyt?

Reply:
Sounds good will do.

Reply:
ping :)

Reply:
pong!